Firmware/ultralcd.cpp

```diff
@@ -5331,10 +5339,16 @@ static void lcd_main_menu()
     if (MMU2::mmu2.Enabled()) {
         MENU_ITEM_SUBMENU_P(_i("Fail stats MMU"), lcd_menu_fails_stats_mmu);////MSG_MMU_FAIL_STATS c=18
     }
+
+#ifdef HOST_SHUTDOWN
+    if (!printer_active() && M79_timer_get_status()) {
+        MENU_ITEM_FUNCTION_P(_T(MSG_SHUTDOWN_HOST), lcd_shutdown_menu);////MSG_SHUTDOWN_HOST c=18
```

Comment:
Please remove `////MSG_SHUTDOWN_HOST c=18`as it this is already in `messages.cpp`

Reply:
Done